grisera_api/participant_state/participant_state_router.py

```diff
@@ -51,38 +53,40 @@ async def create_participant_state(
 
         return create_response
 
+
     @router.get(
         "/participant_state",
         tags=["participant state"],
         response_model=ParticipantStatesOut,
     )
-    async def get_participant_states(self, response: Response):
+    async def get_participant_states(self, response: Response,dataset_name: str):
         """
         Get participant states from database
         """
 
-        get_response = self.participant_state_service.get_participant_states()
+        get_response = self.participant_state_service.get_participant_states(dataset_name)
 
         # add links from hateoas
         get_response.links = get_links(router)
 
         return get_response
 
+
     @router.get(
         "/participant_state/{participant_state_id}",
         tags=["participant state"],
         response_model=Union[ParticipantStateOut, NotFoundByIdModel],
     )
     async def get_participant_state(
-        self, participant_id: Union[int, str], response: Response, depth: int=0
+        self, participant_id: Union[int, str], response: Response,dataset_name: str, depth: int=0
```

Comment:
Please check this formatting in each file.

Reply:
done